### PR TITLE
MISP V2: fixed vendor name

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35832

## Description
fixed the vendor name set in the DBotScore and standard context paths related to MISP indicators.

### CODE REVIEW

**Review by teizenman — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@teizenman @yaakovi @reutshal 
Release notes were reviewed.

**Review by reutshal — COMMENTED** [review]
Do we also want to address this comment here? 
image

File: Packs/MISP/Integrations/MISP_V2/MISP_V2.py [hunk 1]
```diff
@@ -753,7 +753,7 @@ def download_file():
         else:
             file_buffer = response[1][0][2].getbuffer()
             filename = response[1][0][1]
-        demisto.results(fileResult(filename, file_buffer))  # type: ignore
+            demisto.results(fileResult(filename, file_buffer))  # type: ignore
```
[the remark this answers is not in the document]
Comment (yaakovi) [reply Packs/MISP/Integrations/MISP_V2/MISP_V2.py:767]: it's only relevant to the "else" part

**Comment by yaakovi** [comment]
> Do we also want to address this comment here?
> image

done

**Review by teizenman — APPROVED** [review]

**Pull request merged**